### PR TITLE
Use sidecar from dockerhub

### PR DESCRIPTION
**Pull request by mkoderer**
**What this PR does / why we need it**:
Doc fix for sidecar example: For the first try it's easier to download the sidecar from docker hub
instead of compiling it.

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by mkoderer** [comment]
@mfranczy Thanks! Should be fixed now

**Comment by mfranczy** [comment]
ci test please

**Comment by phoracek** [comment]
@mkoderer all the other examples reference to local registry. Also this could serve for hook testing purposes. Don't you need to `make` kubevirt before using the local cluster anyway?

**Comment by mkoderer** [comment]
@phoracek if you deploy a kubevirt release you don't need to compile / make. You simply refer to the existing example-hook-sidecar

File: tools/vms-generator/utils/utils.go [hunk 1]
```diff
@@ -833,7 +833,7 @@ func GetVMIWithHookSidecar() *v1.VirtualMachineInstance {
 	addNoCloudDiskWitUserData(&vmi.Spec, "#cloud-config\npassword: fedora\nchpasswd: { expire: False }")
 
 	vmi.ObjectMeta.Annotations = map[string]string{
-		"hooks.kubevirt.io/hookSidecars":              `[{"image": "registry:5000/kubevirt/example-hook-sidecar:devel"}]`,
+		"hooks.kubevirt.io/hookSidecars":              `[{"image": "kubevirt/example-hook-sidecar:devel"}]`,
```
Comment (davidvossel) [inline tools/vms-generator/utils/utils.go]: I think we're generating this annotation wrong. The example should be using the DockerPrefix value like all the other containers in our generation code. Essentially, this shouldn't have been hardcoded to registry:5000 to begin with.

**Pull request closed without merging**